scripts/update-translations.sh

```diff
@@ -11,5 +11,5 @@ npm run test
 
 # commit any updates and push. Build and release should happen on the push not here.
 git add .
-git commit -m "pull new editor translations from Transifex"
+git commit -m "fix: pull new editor translations from Transifex"
```

Comment:
I wonder if this should be `fix` or `chore` - I assume both will bump the z in x.y.z so the main difference is whether we include "pull new editor translations..." in the release notes every time. To me it feels more in line with updating libraries that fixing bugs.

Reply:
Actually, `fix`, `feat`, and `BREAKING CHANGE` are the only three things that bump version numbers. The `chore` designation puts an entry in the changelog but doesn't cause a version bump or a release, so it gets rolled into the next release.

I've used that distinction in our Renovate config so that updating something `dependencies` bumps the version number and causes a release, but updating something in `devDependencies` (which gets marked as `chore`) doesn't. If we've used those fields correctly, the `devDependencies` update won't be noticeable to the user, so there's no need to release it.

Semver's really designed to care about an API, so pulling new strings (a data update) doesn't really fit the model perfectly. I decided to call it a "fix" because that's the most minor way to bump the version number and cause a release.

Another option: we could add a new designation, like `l10n: pull new strings` or something like that, and configure `semantic-release` to consider that a patch version bump.

Reply:
I do think it would make sense to collapse multiple version bumps for the same library, so the changelog just shows the most recent bump for each library since the last release. It wouldn't surprise me if someone has already written some sort of script or `semantic-release` plugin to do it. So far I've been prioritizing monorepo exploration over fixing up changelogs, though, partly because a monorepo would decrease the `update dependency scratch-*` messages in our changelogs even more than this kind of filter :)